great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -2093,9 +2098,7 @@ def _get_dialect_like_pattern_expression(self, column, like_pattern, positive=Tr
 
         try:
             # Bigquery
-            if isinstance(
-                self.sql_engine_dialect, pybigquery.sqlalchemy_bigquery.BigQueryDialect
-            ):
+            if hasattr(self.sql_engine_dialect, "BigQueryDialect"):
```

Comment:
Could we perhaps log something if we hit an exception? I know logging was causing some issues earlier so avoid if it blocks you. I was just thinking that it might be useful to get some more information than a standard `pass`.

Reply:
Added